test/TestInvestorLedger.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.4.11;
+
+import "truffle/Assert.sol";
+import "truffle/DeployedAddresses.sol";
+import "../contracts/loans/InvestorLedger.sol";
+import "../contracts/tokens/PrintableToken.sol";
+
+
+contract TestInvestorLedger {
+    uint256 totalLoanNeeded = 100;
+    uint16 interestPermil = 200;
+    uint256 EMPTY = 0;
+    uint256 PRINT_VALUE = 100;
```

Comment:
Why the style change between camelCase and ALL_CAPS?

Reply:
forgot to make it consistent. Fixed